blockchain/utxocache.go

```diff
@@ -145,6 +148,8 @@ func (entry *UtxoEntry) memoryUsage() uint64 {
 	//   unsafe.Sizeof(UtxoEntry{})
 	baseEntrySize := uint64(40)
 
+	baseEntrySize += uint64(88) // Size of TokenData
```

Comment:
Where do we get this magic value of 88? Should this be a constant?

Reply:
I don't think a constant is needed. Added a comment on where we got the number from: This value is calculated by running the following on a 64-bit system: unsafe.Sizeof(TokenData{}) assuming commitment of length 40